### DISCUSSION TITLE
Best procedure to update OC 0.6.9 and safely update Big Sur

### DISCUSSION THREAD

**Discussion by spn0381** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#71] [Q&A]
Hi, 

thank you chris1111 for all your work! 

I would like to update my installation of probook Big Sur with the 10 April 2021 Update OC 0.6.9 Update OpenCore Configurator

and after this update macOS Big Sur 11 (20D91). 

Is correct if I make from the Probook a new HP-ProBook-EliteBook-macOS.pkg, and then I will install the package without do anything on EFI?

only after this update I will update macOS Big Sur 11  (20D91). 

thank you in advance for all your help

(doing this update in the previous release - Clover and Catalina, the probook wasn't bootable)

**Comment by chris1111** [comment]
Best procedure is booting from the **HP Probook Create Install media** then remove Folder **OC** on EFI clean the trash
then created and use latest program **HP-ProBook-EliteBook-macOS.pkg**  again
